DesignSafe-CI/portal
# task/DES-2367: Log DOIs when a file operation is performed in a published project

## Overview: ##
When a user performs a listing/preview/download in a published project, log the DOI associated with the file.

## PR Status: ##


## Related Jira tickets: ##

* [DES-2367](https://jira.tacc.utexas.edu/browse/DES-2367)

## Summary of Changes: ##

## Testing Steps: ##
1. Inside of a publication, preview/download a file or list a directory. you should see a log of the form: 
```des_django         | [METRICS] INFO views metrics.get:50: Data Depot user=jarosenb ip=172.20.0.1 agent=Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/115.0.0.0 Safari/537.36 sessionId=szaq582dydm3w08fnv622xpfkw8pr1uu op=preview info={'api': 'agave', 'systemId': 'designsafe.storage.published', 'filePath': 'PRJ-2470/CWU data/UW_R_Scripts/Stanley/data analysis_wind speed.ipynb', 'doi': '10.17603/ds2-e2av-nx29', 'query': {'doi': '10.17603/ds2-e2av-nx29', 'href': ''}}```

The important thing is the `doi` field.
